DigitalLearningSolutions.Data/DataServices/UserDataService/DelegateUserDataService.cs

```diff
@@ -140,7 +140,8 @@ public List<DelegateUser> GetDelegateUsersByEmailAddress(string emailAddress)
                         cd.Answer5,
                         cd.Answer6,
                         cd.JobGroupID,
-                        jg.JobGroupName
+                        jg.JobGroupName,
+                        cd.ResetPasswordID

```

Comment:
This change leads me to think that the original code for invalidating hashes did not work as expected as it did not recover the delegate's reset password Ids. Are there any unit tests we could implement that would have spotted the issue? I can only think that the unit tests for this method and the admin equivalent should retrieve data that has a non-null reset password id. Thoughts?

Reply:
Good question, it did look to me like the original code didn't work for delegate hashes. I only discovered this problem when manually testing. I'll take another look at the tests, I recall I tried to find a quick fix and failed, but it's worth fixing.